src/main/java/uk/gov/hmcts/befta/dse/ccd/definition/converter/JsonTransformer.java

```diff
@@ -73,6 +74,12 @@ public void parseDefinitionJson(String path) {
                 String sheet = null;
                 try {
                     JsonNode rootSheetArray = objectMapper.readTree(jsonFile);
+
+                    if (BeftaMain.getConfig().getTestUrl().contains("localhost")) {
+                        String rootSheetArrayString = rootSheetArray.toString().replaceAll("ccd-test-stubs-service-aat.service.core-compute-aat.internal", "ccd-test-stubs-service:5555");
+                        rootSheetArray = objectMapper.readTree(rootSheetArrayString);
+                    }
```

Comment:
This is doing the trick to all the files, and perhaps is an inefficient implementation as such. Could we please apply this trick only to some specific json files corresponding to some specific worksheets in the Excel file?

Reply:
Addressed in https://github.com/hmcts/befta-fw/pull/93/commits/7727e368552565070660acfa0573d34543570f48